src/test/unit_tests/jest_tests/test_nc_nsfs_account_cli.test.js

```diff
@@ -1765,6 +1861,47 @@ describe('manage nsfs cli account flow', () => {
             const res = await exec_manage_cli(type, action, command_flags);
             expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidAccountName.code);
         });
+
+        it('cli create account using from_file with supplemental_groups', async () => {
+            const action = ACTIONS.ADD;
+            const { name, new_buckets_path, uid, gid } = defaults;
+            const supplemental_groups = [212, 112];
+            const account_options = { name, new_buckets_path, uid, gid, supplemental_groups };
+            // write the json_file_options
+            const path_to_option_json_file_name = await create_json_account_options(path_to_json_account_options_dir, account_options);
+            const command_flags = {config_root, from_file: path_to_option_json_file_name};
+            // create the account and check the details
+            await exec_manage_cli(type, action, command_flags);
+            // compare the details
+            const account = await config_fs.get_account_by_name(name, config_fs_account_options);
+            expect(account.nsfs_account_config.supplemental_groups).toStrictEqual(supplemental_groups);
+        });
+
+        it('should fail - cli create account using from_file with invalid supplemental_groups (negative number)', async () => {
+            const action = ACTIONS.ADD;
+            const { name, new_buckets_path, uid, gid } = defaults;
+            const supplemental_groups = [212, -112]; // gid cannot be negative number
+            const account_options = { name, new_buckets_path, uid, gid, supplemental_groups };
+            // write the json_file_options
+            const path_to_option_json_file_name = await create_json_account_options(path_to_json_account_options_dir, account_options);
+            const command_flags = {config_root, from_file: path_to_option_json_file_name};
+            // create the account and check the details
+            const res = await exec_manage_cli(type, action, command_flags);
+            expect(JSON.parse(res.stdout).error.code).toBe(ManageCLIError.InvalidSupplementalGroupsList.code);
+        });
+
+        it('should fail - cli create account using from_file with invalid supplemental_groups (string)', async () => {
+            const action = ACTIONS.ADD;
+            const { name, new_buckets_path, uid, gid } = defaults;
+            const supplemental_groups = ["212d", 112]; // gid cannot a non number type
```

Comment:
Please change single quotes to double quotes.
```suggestion
            const supplemental_groups = ['212d', 112]; // gid cannot a non number type
```